packages/api-explorer/src/index.jsx

```diff
@@ -197,7 +209,9 @@ class ApiExplorer extends React.Component {
                         oas={this.getOas(doc)}
                         oauth={this.props.oauth}
                         onAuthChange={this.onAuthChange}
+                        onDocRender={this.onDocRender}
                         onGroupChange={this.onGroupChange}
+                        rendered={this.state.docRenderMap[doc._id]}
```

Comment:
Instead of keying this off the doc id, can we use `doc.slug` instead? I'd love for us to reduce our Mongo model properties in here, and slug should be unique.

Reply:
Works for me!

Reply:
@domharrington are the slugs sufficiently unique for this use case?